plone/app/dexterity/behaviors/metadata.py

```diff
@@ -2,9 +2,9 @@
 from datetime import datetime
 from DateTime import DateTime
 from plone.app.dexterity import _
-from plone.app.z3cform.widget import AjaxSelectFieldWidget
-from plone.app.z3cform.widget import DatetimeFieldWidget
-from plone.app.z3cform.widget import SelectFieldWidget
+from plone.app.z3cform.widgets.datetime import DatetimeFieldWidget
+from plone.app.z3cform.widgets.select import AjaxSelectFieldWidget
+from plone.app.z3cform.widgets.select import SelectFieldWidget
```

Comment:
The correct import for the former `SelectFieldWidget` is now `.widgets.select.Select2FieldWidget` ... see https://github.com/plone/plone.app.z3cform/blob/master/plone/app/z3cform/widget.py#L54 ... the `SelectFieldWidget` is a bare Bootstrap5 flavored `<select />`